server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1436,6 +1441,89 @@ public void run() {
     log.info("exiting");
   }
 
+  /**
+   * Allows property configuration to block master start-up waiting for a minimum number of tservers
+   * to register in zookeeper. It also accepts a maximum time to wait - if the time expires, the
+   * start-up will continue with any tservers available. The following properties are used to
+   * control the behaviour:
+   * <ul>
+   * <li>MASTER_STARTUP_TSERVER_AVAIL_MIN_COUNT - when set to 0, no blocking occurs (default
+   * behaviour) otherwise will block until the number of tservers are available.</li>
+   * <li>MASTER_STARTUP_TSERVER_AVAIL_MAX_WAIT - time to wait in milliseconds. When set to 0, will
+   * block indefinitely.</li>
```

Comment:
When set to 0 or less, will block indefinitely until the number of tservers are available.